Code/Framework/AzCore/AzCore/Debug/Trace.cpp

```diff
@@ -110,11 +165,40 @@ namespace AZ::Debug
             g_ignoredAsserts = AZ::Environment::CreateVariable<AZStd::unordered_set<size_t>>(ignoredAssertUID);
             g_assertVerbosityLevel = AZ::Environment::CreateVariable<int>(assertVerbosityUID);
             g_logVerbosityLevel = AZ::Environment::CreateVariable<int>(logVerbosityUID);
+            s_fileStream = AZ::Environment::CreateVariable<FILE*>(fileStreamIdentifier, stdout);
 
             //default assert level is to log/print asserts this can be overriden with the sys_asserts CVAR
             g_assertVerbosityLevel.Set(assertLevel_log);
             g_logVerbosityLevel.Set(logLevel_full);
         }
+
+        // Setup the raw C FILE* pointer to allow raw output to write to one of the std streams
+        s_fileStream = AZ::Environment::FindVariable<FILE*>(fileStreamIdentifier);
+        if (!s_fileStream)
+        {
+            FILE* redirectStream = stdout;
+            if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
+            {
+                if (RedirectCStream redirectOption;
+                    console->GetCvarValue("bg_redirectrawoutput", redirectOption) == AZ::GetValueResult::Success)
+                {
+                    switch (redirectOption)
+                    {
+                    case RedirectCStream::Stdout:
+                        redirectStream = stdout;
+                        break;
+                    case RedirectCStream::Stderr:
+                        redirectStream = stderr;
+                        break;
+                    case RedirectCStream::None:
+                        redirectStream = nullptr;
```

Comment:
would be nice to have unit tests for Stderr and None modes if we don't have them already

Reply:
Yeah I'll add a test for it.